packages/grafana-ui/src/utils/standardEditors.tsx

```diff
@@ -190,6 +193,30 @@ export const getStandardFieldConfigs = () => {
     category,
   };
 
+  const nullValue: FieldConfigPropertyItem<any, NullValueMode, SelectFieldConfigSettings<NullValueMode>> = {
+    id: 'nullValueMode',
+    path: 'nullValueMode',
+    name: 'Null values',
+    description: 'Behavior for null values in the data',
+
+    editor: standardEditorsRegistry.get('radio').editor as any,
+    override: standardEditorsRegistry.get('radio').editor as any,
+    process: selectOverrideProcessor,
+
+    settings: {
+      options: [
+        { label: 'Null', value: NullValueMode.Null },
+        // The "connectd" of "span gaps" behavior is moved to graph settings.
+        // This option should either be a boolean or removed an just "AzZero" added as a transformer
+        // { label: 'Ignore', value: NullValueMode.Ignore, description: 'Ignore the values from any calculation' },
```

Comment:
I'm hiding the "ignore" option since that has a less clear migration path.   For "AzZero" we can use a transformation